### PR TITLE
kde-inhibit: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/kde-inhibit.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+`kde-inhibit --power {{command}} {{command_arguments}}`
+
+- Inhibit screen saver:
+
+`kde-inhibit --screenSaver {{command}} {{command_arguments}}`
+
+- Inhibit colour correction (night mode) while using VLC:
+
+`kde-inhibit --colorCorrect {{vlc}}`
```
Comment (sbrl) [inline pages/linux/kde-inhibit.md:15]: Does this save the preference to a file, or run the command itself? Given that this example is slightly different to the previous 2, I'm a little unsure and just want to check :-)

Comment (navarroaxel) [reply pages/linux/kde-inhibit.md:15]: No, only inhibit the `night mode` if you start `vlc` using this command, and start the `night mode` when `vlc` exists. It's just a very concrete example.

Comment (sbrl) [reply pages/linux/kde-inhibit.md:15]: Ah, I see. Thanks!

File: pages/linux/kde-inhibit.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Inhibit power management:
+
+`kde-inhibit --power {{command}} {{command_arguments}}`
+
+- Inhibit screen saver:
+
+`kde-inhibit --screenSaver {{command}} {{command_arguments}}`
+
+- Inhibit colour correction (night mode) while using VLC:
```
Comment (sbrl) [inline pages/linux/kde-inhibit.md:13]: What about this?
```suggestion
- Launch vlc, and inhibit colour correction (night mode) while it's running:
```

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel :heart:

**Comment by navarroaxel** [comment]
@owenvoke or @schneiderl could you please check?

**Review by owenvoke — APPROVED** [review]

**Pull request merged**